libraries/DataFlash/DataFlash_File.cpp

```diff
@@ -1025,7 +1025,9 @@ void DataFlash_File::flush(void)
            BUF_AVAILABLE(_writebuf)) {
         // convince the IO timer that it really is OK to write out
         // less than _writebuf_chunk bytes:
-        _last_write_time = tnow - 2000000;
+        if (tnow > 2000001) {
```

Comment:
On Tue, 26 Apr 2016, Lucas De Marchi wrote:

> In libraries/DataFlash/DataFlash_File.cpp:
> 
> > @@ -1025,7 +1025,9 @@ void DataFlash_File::flush(void)
> >             BUF_AVAILABLE(_writebuf)) {
> >          // convince the IO timer that it really is OK to write out
> >          // less than _writebuf_chunk bytes:
> > -        _last_write_time = tnow - 2000000;
> > +        if (tnow > 2000001) {
> 
> Should be >= ?

I stared at that and decided to avoid resetting it to 0, just in case 
someone decides to use it as a flag value in future.  1us here or there 
doesn't matter in this loop :-)  Will add a comment.